cmd/tracee-ebpf/main.go

```diff
@@ -88,11 +89,11 @@ func main() {
 
 			// environment capabilities
 
-			selfCap, err := getSelfCapabilities()
+			selfCap, err := capabilities.SelfCapabilities()
```

Comment:
I'm wondering if we could avoid repetition in package names and exported types / functions as explained in [Effective Go Package Names](https://go.dev/doc/effective_go#package-names) WDYT?

```suggestion
			selfCap, err := capabilities.Self()
```